ingestion-beam/src/main/java/com/mozilla/telemetry/schemas/SchemaAlias.java

```diff
@@ -0,0 +1,37 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package com.mozilla.telemetry.schemas;
+
+import static org.apache.commons.lang3.builder.ToStringStyle.SHORT_PREFIX_STYLE;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+
+class SchemaAlias {
```

Comment:
Slight preference for making this an AutoValue class to reduce some boilerplate and make this more consistent with AutoValue usage elsewhere in this codebase and in Beam.

Reply:
Nice, I didn't know about AutoValue.
I thought about Lombok when generating this, but didn't want to introduce this library here.